### PR TITLE
Extract settings for VR module into ini file

### PR DESCRIPTION
**Pull request by bonewell**

### CODE REVIEW

**Comment by KKarlash** [comment]
Reviewed

File: src/components/vr_module/src/socket_channel.cc [hunk 1]
```diff
@@ -54,9 +60,7 @@ SocketChannel::~SocketChannel() {
 
 bool SocketChannel::Start() {
   LOG4CXX_AUTO_TRACE(logger_);
-  std::string address = "127.0.0.1";
-  UInt32 port = 5431;
-  socket_ = net::ConnectedSocketImpl::ConnectToHost(address.c_str(), port);
+  socket_ = net::ConnectedSocketImpl::ConnectToHost(address_.c_str(), port_);
   return false;
```
Comment (DKlimenko) [inline src/components/vr_module/src/socket_channel.cc]: we will always return false?

**Comment by DKlimenko** [comment]
reviewed

File: src/components/vr_module/src/socket_channel.cc (hunk 1 above)
Comment (bonewell) [inline src/components/vr_module/src/socket_channel.cc]: I have fixed it in another pull request which is under review now.

Comment (VladSemenyuk) [inline src/components/vr_module/src/socket_channel.cc]: sometimes it is good to return false always, I believe honestly.))))

**Comment by VladSemenyuk** [comment]
Reviewed

**Pull request merged**